### PR TITLE
Creating Day1- Ex03

### PR DESCRIPTION
**Pull request by mbrigida**
* Created Exercise003
* Created Exercise003.solutions

### CODE REVIEW

File: MMUL-BASH-01/Exercises/Day-1/Exercise.003 [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Exercise 003 - Redirections
+
+1) Log into the machine with the credentials you own;
+
+2) print the string "# Exercise 003 - Redirect STDOUT" to a file called
```
Comment (rascasoft) [inline MMUL-BASH-01/Exercises/Day-1/Exercise.003]: Start with capital letter each section, you're mixing capital and lowercase.

File: MMUL-BASH-01/Exercises/Day-1/Exercise.003 [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Exercise 003 - Redirections
+
+1) Log into the machine with the credentials you own;
+
+2) print the string "# Exercise 003 - Redirect STDOUT" to a file called 
+"output_data" in your HOMEDIR;
```
Comment (rascasoft) [inline MMUL-BASH-01/Exercises/Day-1/Exercise.003]: Align the second line with the previous text, like:
```
1) First line starts here
   Second starts here.
```

**Review by rascasoft — CHANGES_REQUESTED** [review]
Few changes and we're good to go.

**Comment by mbrigida** [comment]
Capitalization at start of each section, text aligned with previous text

**Review by rascasoft — APPROVED** [review]
Looks good to me.

**Pull request merged**